npm-shrinkwrap.json

```diff
@@ -366,78 +380,78 @@
     },
     "nodemailer": {
       "version": "1.3.0",
-      "from": "nodemailer@^1.1.1",
+      "from": "https://registry.npmjs.org/nodemailer/-/nodemailer-1.3.0.tgz",
       "resolved": "https://registry.npmjs.org/nodemailer/-/nodemailer-1.3.0.tgz",
       "dependencies": {
         "buildmail": {
           "version": "1.2.0",
-          "from": "buildmail@^1.2.0",
+          "from": "https://registry.npmjs.org/buildmail/-/buildmail-1.2.0.tgz",
           "resolved": "https://registry.npmjs.org/buildmail/-/buildmail-1.2.0.tgz",
           "dependencies": {
             "addressparser": {
               "version": "0.3.2",
-              "from": "addressparser@^0.3.1",
+              "from": "https://registry.npmjs.org/addressparser/-/addressparser-0.3.2.tgz",
```

Comment:
After a second look at this, `version` and `resolved` haven't changed, so it seems to be ok. NTC.